util/request_source.go

```diff
@@ -40,6 +53,9 @@ const (
 type RequestSource struct {
 	RequestSourceInternal bool
 	RequestSourceType     string
+	// ExplicitRequestSourceType is set from the session variable, it may specified by the client or users. `explicit_request_source_type`. it's a complement of RequestSourceType.
```

Comment:
```suggestion
	// ExplicitRequestSourceType is set from the session variable, it may specified by the client or users with `tidb_request_source_type `. 
	// It's a complement of RequestSourceType.
```